src/main/java/Bao.java

```diff
@@ -0,0 +1,116 @@
+import Components.Task;
+import Components.Todo;
+import Components.Deadline;
+import Components.Event;
+
+import java.util.Scanner;
+
+public class Bao {
+    private static Scanner in = new Scanner(System.in);
+    private static Task[] tasks = new Task[100];
+    private static int numTasks=0;
+
```

Comment:
Any reason why you did not include a variable separator so you do not have to print the long line of underscore?
```suggestion
private static String separator = "______________________________________________________________________________________";
```